deploy/eck-integrations/README.md

```diff
@@ -0,0 +1,58 @@
+# ECK-Integrations
+
+ECK Integrations is a Helm chart to assist in the deployment of Elastic integrations, which are
+managed by the [ECK Operator](https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html)
+
+## Supported Elastic Integrations
+
+The following Elastic integrations are currently supported. 
+
+- Kubernetes
```

Comment:
Cloud Defend?

Reply:
yes you can set the cloudDefend.enabled = true helm chart flag but as of now it only adjusts and redeploys the agent of the daemonset with the appropriate mounts and permissions